app/download/views.py

```diff
@@ -93,3 +98,40 @@ def download_document_b64(service_id, document_id):
     response.headers["X-Robots-Tag"] = "noindex, nofollow"
 
     return response
+
+
+@download_blueprint.route(
+    "/services/<uuid:service_id>/documents/<uuid:document_id>/scan-verdict", methods=["POST"]
+)
+def check_scan_verdict(service_id, document_id):
+    sending_method = request.form.get("sending_method")
+    try:
+        av_status = scan_files_document_store.check_scan_verdict(service_id, document_id, sending_method)
+    except (MaliciousContentError, SuspiciousContentError) as e:
+        current_app.logger.info(
+            "Malicious content detected, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), 403
```

Comment:
We could have these error code hardcoded in constants at the top of the module, to easily keep track and not duplicate.